### PR TITLE
Implement a basic main game panel.

### PR DESCRIPTION
**Pull request by yayixu**

### CODE REVIEW

File: src/TetrisGame/BoardedPanel.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+public class BoardedPanel extends JPanel {
+
+    @Override
+    public void paintComponent(Graphics g) {
+        super.paintComponent(g);
+        Rectangle rec = this.getBounds();
+        g.setColor(Color.black);
+        g.drawRect(0, 0, rec.width, rec.height);
+    }
+
```
Comment (jollycopper) [inline src/TetrisGame/BoardedPanel.java:18]: Nit: remove phantom empty line.

File: src/TetrisGame/GamePanel.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.Set;
+/**
+ * Created by yayixu on 9/10/17.
+ */
+public class GamePanel extends BoardedPanel{
+    public static final int UNIT = 50;
+    private static final int WIDTH = 5, HEIGHT = 10;
+    private static final String PAUSE = "PAUSE";
+
+    public static int convert(double x) {
```
Comment (jollycopper) [inline src/TetrisGame/GamePanel.java]: Nit: method should always be placed after attributes.

File: src/TetrisGame/GamePanel.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    public GamePanel() {
+        setLayout(null);
+        setBackground(Color.white);
+        // Added a pause button into Game Panel
+        pauseButton = new JButton(PAUSE);
+
+        add(pauseButton);
+        pauseButton.setVisible(false);
+        pauseButton.setBounds(70, 225, 100, 50);
+        invalidate();
```
Comment (jollycopper) [inline src/TetrisGame/GamePanel.java]: I don't think you need this.

File: src/TetrisGame/TetrisApp.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * Created by yayixu on 9/10/17.
+ */
+import java.awt.*;
+import javax.swing.*;
+
+public class TetrisApp {
+    private static final String APPNAME = "yayiTetrisGame";
+    private static final int WIDTH = 600, HEIGHT = 600;
+
+
```
Comment (jollycopper) [inline src/TetrisGame/TetrisApp.java]: Nit: empty line

File: src/TetrisGame/BoardedPanel.java (hunk 1 above)
Comment (yayixu) [reply src/TetrisGame/BoardedPanel.java:18]: Done.

File: src/TetrisGame/GamePanel.java (hunk 2 above)
Comment (yayixu) [reply src/TetrisGame/GamePanel.java]: Done.

File: src/TetrisGame/GamePanel.java (hunk 3 above)
Comment (yayixu) [reply src/TetrisGame/GamePanel.java]: Done.

File: src/TetrisGame/TetrisApp.java (hunk 4 above)
Comment (yayixu) [reply src/TetrisGame/TetrisApp.java]: Done.

**Review by jollycopper — APPROVED** [review]
GJ!

**Pull request merged**